src/message_transfer/message_transfer.go

```diff
@@ -26,15 +26,15 @@ func (messageTransfer *MessageTransfer) Broadcast(update dataTransferModels.Pack
 	messageTransfer.channel <- message
 }
 
-func getMessage(update dataTransferModels.PacketUpdate) models.Message {
+func getMessage(update board_models.Update) models.Message {
 	var message models.Message
-	if msg, ok := update.Values["warning"]; ok {
+	if msg, ok := update.Fields["warning"]; ok {
 		message = models.Message{
 			ID:          update.ID,
 			Description: msg.(string),
 			Type:        "warning",
 		}
-	} else if msg, ok = update.Values["fault"]; ok {
+	} else if msg, ok = update.Fields["fault"]; ok {
```

Comment:
Add else with an error message of incorrect message type.

Reply:
And by the way, change the else if to a non-indented if (there are better ways to do this but it's not worth implementing at this point)